packages/flutter/lib/semantics.dart

```diff
@@ -13,6 +13,8 @@
 /// and is used by the platform-specific accessibility services.
 library semantics;
 
+export 'dart:ui' show LocaleStringAttribute, SpellOutStringAttribute;
```

Comment:
It's a good option to export here (a file with only exports) because the lint ignores such a file when it checks the exported types are used in the public API.